webpack.config.js

```diff
@@ -108,11 +108,15 @@ module.exports = {
         options: {
           name: '[name].[ext]',
           outputPath: 'fonts/',
-          publicPath: 'bundles/'
+          publicPath: 'bundles/fonts/'
         }
       }
     ]
   },
 
+  optimization: {
+    minimize: false
```

Comment:
By default, production webpack builds now run UglifyJS`.  Unfortunately, Uglify is failing on our `updater.js` entry with an extremely helpful error message of `undefined`.  I'll look into it later, but minification is of relatively limited value to us since we're not trying obfuscate code, and we're not loading it over the web on page load.